packages/arb-validator-core/gobridge/challenge.go

```diff
@@ -30,39 +31,45 @@ const (
 	challengerTurn = 2
 )
 
+type challengeData struct {
+	deadline             common.TimeTicks
+	challengerDataHash   common.Hash
+	state                int
+	challengePeriodTicks common.TimeTicks
+	asserter             common.Address
+	challenger           common.Address
+	challengeType        *big.Int
+}
+
 type challenge struct {
 	client          *GoArbAuthClient
 	challengeData   *challengeData
 	contractAddress common.Address
 }
```

Comment:
Is there a reason `challengeData` is a separate struct instead of those fields just being part of `challenge`?

Reply:
I thought it looked cleaner, maybe I will add challengeData as an anonymous field in challenge to make referencing the data cleaner.

Reply:
I don't think it's really worth keeping them separate. There are only two other fields in the `challenge` struct so the size would barely change.

Reply:
fixed